charts/pega/charts/installer/templates/_helpers.tpl

```diff
@@ -54,13 +54,20 @@
   image: {{ .Values.global.utilityImages.k8s_wait_for.image }}
   imagePullPolicy: {{ .Values.global.utilityImages.k8s_wait_for.imagePullPolicy }}
   args: [ 'job', '{{ template "pegaDBInstall" }}']
+  env:  
+    - name: WAIT_TIME
+      value: "{{ .Values.global.utilityImages.k8s_wait_for.waitTimeSeconds }}"
```

Comment:
yeah I feel like adding a default here would be cleaner. Mainly because of the question "how does one know about the default in the base image?". Even if we set default to be same as base image, at least you'd be able to see here *what* the default actually is